main.py

```diff
@@ -0,0 +1,70 @@
+import random
+
+
+class Trip:
+    def __init__(self, traveler_name, traveler_age):
+        self.traveler_name = traveler_name
+        self.traveler_age = traveler_age
+        self.citycost = {
+            "kyiv": 890,
+            "kharkiv": 723,
+            "odessa": 472,
+            "poltava": 177,
+            "dnipro": 200,
+            "lviv": 403
+        }
+        print "Please input amount of nights"
+        self.nights = input()
+
+        print "Please choose one of the city: Kyiv, Kharkiv, Odessa, Poltava, Dnipro, Lviv"
+        self.city = raw_input().lower()
+
+    def plane_city_cost(self):
+        plane_city_amount = 0
+        if self.city in self.citycost:
+            k = self.city
+            plane_city_amount = self.citycost.get(k)
+            return plane_city_amount
+        else:
+            print "Sorry, No matches found"
+            return plane_city_amount
+
+    print "The plane ride cost", plane_city_cost()
+
+    def hotel_cost(self):
+        hotel_amount = 140 * self.nights
+        return hotel_amount
+
+    print "The hotel cost is", hotel_cost()
```

Comment:
ok, this will not work as you might expect: when you put `print` like this the method `hotel_cost` will be callable because they are both in the class scope (please, read on `variables and scope in Python`), but that method will expect one argument (named as `self`) and you are not providing anything to it

So once again, you should use printing outside the class smth like
`print('The hotel cost is', first.hotel_cost())`